website_sale_require_legal/views/website_sale.xml

```diff
@@ -1,23 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+<!-- Copyright 2017 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
+     Copyright 2019 Tecnativa - Cristina Martin R
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
 
     <template id="accept_input">
-        <div t-attf-class="form-group col-md-12 #{'has-error' if error and error.get('accepted_legal_terms') else ''}">
-            <label for="accepted_legal_terms" class="control-label">
-                <input
-                    type="checkbox"
-                    name="accepted_legal_terms"
-                    id="accepted_legal_terms"
-                    required="required"/>
+        <div t-attf-class="form-group col-lg-12">
```

Comment:
If this has no changes, there's no need to use `t-attf-`. Remove that part please.